### ISSUE TITLE
"travis" command-line tool complains "private method `load' called for nil:NilClass"

### ISSUE THREAD

**Issue by jameshfisher** [travis-ci/travis.rb#248]
Whenever I run any `travis` commands I get this:

```
$ travis setup releases
private method `load' called for nil:NilClass
for a full error report, run travis report --pro
```

That command gives me:

```
$ travis report --pro
System
Ruby:                     Ruby 1.9.3-p484
Operating System:         Ubuntu 14.04
RubyGems:                 RubyGems 1.8.23

CLI
Version:                  1.7.4
Plugins:                  none
Auto-Completion:          yes
Last Version Check:       2014-11-23 17:50:02 +0000

Session
API Endpoint:             https://api.travis-ci.com/
Logged In:                as "jameshfisher"
Verify SSL:               yes
Enterprise:               no

Endpoints
pro:                      https://api.travis-ci.com/ (access token, current)

Last Exception
An error occurred running `travis encrypt --pro`:
    NoMethodError: private method `load' called for nil:NilClass
        from /var/lib/gems/1.9.1/gems/travis-1.7.4/lib/travis/cli/repo_command.rb:25:in `setup'
        from /var/lib/gems/1.9.1/gems/travis-1.7.4/lib/travis/cli/command.rb:197:in `execute'
        from /var/lib/gems/1.9.1/gems/travis-1.7.4/lib/travis/cli.rb:64:in `run'
        from /var/lib/gems/1.9.1/gems/travis-1.7.4/bin/travis:18:in `<top (required)>'
        from /usr/local/bin/travis:23:in `load'
        from /usr/local/bin/travis:23:in `<main>'
```

**Comment by reid-spencer** [comment]
+1

**Comment by reid-spencer** [comment]
travis report --pro
System
Ruby:                     Ruby 2.0.0-p481
Operating System:         Mac OS X 10.10.3
RubyGems:                 RubyGems 2.0.14

CLI
Version:                  1.7.7
Plugins:                  none
Auto-Completion:          yes
Last Version Check:       2015-06-07 00:21:14 -0400

Session
API Endpoint:             https://api.travis-ci.com/
Logged In:                as "reid-spencer"
Verify SSL:               yes
Enterprise:               no

Endpoints
org:                      https://api.travis-ci.org/ ()
pro:                      https://api.travis-ci.com/ (access token, current)

Last Exception
An error occurred running `travis encrypt --pro`:
    NoMethodError: private method `load' called for nil:NilClass
        from /Library/Ruby/Gems/2.0.0/gems/travis-1.7.7/lib/travis/cli/repo_command.rb:25:in `setup'
        from /Library/Ruby/Gems/2.0.0/gems/travis-1.7.7/lib/travis/cli/command.rb:197:in `execute'
        from /Library/Ruby/Gems/2.0.0/gems/travis-1.7.7/lib/travis/cli.rb:64:in `run'
        from /Library/Ruby/Gems/2.0.0/gems/travis-1.7.7/bin/travis:18:in `<top (required)>'
        from /usr/bin/travis:23:in `load'
        from /usr/bin/travis:23:in `<main>'

**Comment by mikeljames** [comment]
Hi I'm getting this error only on one project. I have another travis project setup thats currently working. 

I am getting this error when I run most commands such as:

`travis encrypt foo=bar --add`  `travis cache` `travis init`

but when I change directory into another project it works fine...

**Comment by coryellenberger** [comment]
Hello, I am also having difficulty with this problem. Something I've noticed common for all of us is we are using private Repo's and private travis instances.

I'm really only doing this so I can move my repo to public; I will make it public and give it another shot and let you know..

System
Ruby:                     Ruby 2.0.0-p647
Operating System:         Windows
RubyGems:                 RubyGems 2.0.14.1

CLI
Version:                  1.8.0
Plugins:                  none
Auto-Completion:          yes
Last Version Check:       never

Session
API Endpoint:             https://api.travis-ci.com/
Logged In:                as "coryellenberger"
Verify SSL:               yes
Enterprise:               no

Endpoints
pro:                      https://api.travis-ci.com/ (access token, current)
org:                      https://api.travis-ci.org/ (access token)

Last Exception
An error occurred running `travis encrypt --pro`:
    NoMethodError: private method `load' called for nil:NilClass
        from C:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/travis-1.8.0/lib/travis/cli/repo_command.rb:25:in `setup'
        from C:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/travis-1.8.0/lib/travis/cli/command.rb:197:in `execute'
        from C:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/travis-1.8.0/lib/travis/cli.rb:64:in `run'
        from C:/Ruby200-x64/lib/ruby/gems/2.0.0/gems/travis-1.8.0/bin/travis:18:in `<top (required)>'
        from C:/Ruby200-x64/bin/travis:23:in `load'
        from C:/Ruby200-x64/bin/travis:23:in `<main>'

**Comment by mikeljames** [comment]
@coryellenberger I've also moved this to a public repo.

**Comment by coryellenberger** [comment]
@mikeljames I've transitioned to public and am still having this problem. I'm going to attempt reinstalling, etc and see if that works. If not then I'll try a new heroku instance from scratch and see if that helps

**Comment by mikeljames** [comment]
@coryellenberger yeah its not the cli install its some state being managed incorrectly for that one project, I was able to `travis init` within a new repo for a different project.

**Comment by coryellenberger** [comment]
@mikeljames thank you :) that will save me some time.

**Comment by coryellenberger** [comment]
@mikeljames Not sure this will be helpful for you.

But what I ended up doing was this;

I created an empty directory.
Cloned my github repository, cd into my project folder
$travis login
input credentials
$travis settings (this seemed to pull back my configuration)
confirmed my repository
$heroku login
input credentials
$heroku auth:token (to confirm I was getting the correct token back)
$travis encrypt $(heroku auth:token) -a
(Success)

In my case a new directory with a fresh clone and login seemed to work.
